lang/luasocket5.3/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2009-2013 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luasocket
+PKG_SOURCE_VERSION:=733af884f1aa18ff469bf3c4d18810e815853211
+PKG_VERSION:=3.0-rc1-20190421
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_MIRROR_HASH:=915b06219cb79802fe547c1b1896bd7c5465b717549d6a0f38e2c0b4f3e7ae3e
+PKG_SOURCE_URL:=https://github.com/diegonehab/luasocket.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/luasocket5.3
+  SUBMENU:=Lua
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=LuaSocket 5.3
+  URL:=http://luasocket.luaforge.net/
+  MAINTAINER:=W. Michael Petullo <mike@flyn.org>
```

Comment:
Is @flyn-org aware of it?

Reply:
I wasn't sure about this point, since it is an updated package.  I asked one of the other opentwrt developers and they suggested submitting the merge request with the current package maintainer.  I simply wasn't sure how to approach this.

Reply:
The original package also needs similar cleanup.

Reply:
@neheb - how do I go about resolving this (I don't contribute a lot of open source so I don't know all the rules and etiquette).  Do I e-mail the maintainer of the original luasocket package and ask them to maintain this one?  Am I supposed to be maintainer of this new package even though it is a copy of their package?  I apologize for being ignorant on this.

Reply:
Whatever you want. Since you're submitting a new package, I welcome you to maintain it. Or leave it empty if you do not want to.

Reply:
@neheb - I was unaware it was an option to leave maintainer empty.  I've done that.